tests/phpunit/tests/import/import.php

```diff
@@ -18,7 +18,7 @@ function setUp() {
 		add_filter( 'import_allow_create_users', '__return_true' );
 
 		if ( ! file_exists( DIR_TESTDATA . '/plugins/wordpress-importer/wordpress-importer.php' ) ) {
-			$this->fail( 'This test requires the WordPress Importer plugin to be installed in the test suite. See: https://make.wordpress.org/core/handbook/contribute/git/#unit-tests' );
+			$this->fail( 'This test requires the ClassicPress Importer plugin to be installed in the test suite. See: https://make.wordpress.org/core/handbook/contribute/git/#unit-tests' );
```

Comment:
ClassicPress Importer plugin has not yet been developed, maybe we should leave this as is for now? If we change it, shouldn't the link on L20 then not also change?